semanticdb/metap/src/main/scala/scala/meta/cli/Metap.scala

```diff
@@ -10,4 +10,4 @@ object Metap {
   def process(args: Array[String]): Int = {
     Main.process(args)
   }
-}
\ No newline at end of file
+}
```

Comment:
It's regrettable that I sometimes forget adding newlines at the end of files. No idea why I haven't enabled this by default in Sublime. On a related note, can scalafmt take care of that?

Reply:
Scalafmt does not yet do that to my knowledge but you can add an editorconfig (http://editorconfig.org) file to request trailing new lines in most editors